test/framework/easyblock.py

```diff
@@ -1380,6 +1384,11 @@ def test_parallel(self):
         test_eb.check_readiness_step()
         self.assertEqual(test_eb.cfg['parallel'], 67)
 
+        # make sure 'parallel = False' is not overriden
+        test_eb = EasyBlock(EasyConfig(toy_ec3))
+        test_eb.check_readiness_step()
+        self.assertEqual(test_eb.cfg['parallel'], False)
```

Comment:
@migueldiascosta Looks good, but you should also do the same test in the context where `--parallel` is not used (i.e. copy-paste this stuff above line 1372 where `--parallel 97` is put in place, to ensure `parallel = False` works as expected in both contexts.

Side-note: this test only passes because `0 == False` evaluates to `True` in Python...

When `--parallel` is also specified, the current codes takes the minimum of that value (`97` in the test) and the `int` values of the `parallel` easyconfig parameter; with `int(False)` returning `0`, you get `min(97, 0)` or just `0`.

That's not necessarily a problem though, but maybe we should check for `0` explicitly here instead?
We should keep checking against `False` in the context where `--parallel` is not set though...